docs/html/reference/pip_install.rst

```diff
@@ -331,32 +332,35 @@ VCS Support
 +++++++++++
 
 pip supports installing from Git, Mercurial, Subversion and Bazaar, and detects
-the type of VCS using url prefixes: "git+", "hg+", "bzr+", "svn+".
+the type of VCS using url prefixes: ``git+``, ``hg+``, ``svn+``, and ``bzr+``.
 
-pip requires a working VCS command on your path: git, hg, svn, or bzr.
+pip requires a working VCS command on your path: ``git``, ``hg``, ``svn``, or
+``bzr``.
 
 VCS projects can be installed in :ref:`editable mode <editable-installs>` (using
 the :ref:`--editable <install_--editable>` option) or not.
 
-* For editable installs, the clone location by default is "<venv
-  path>/src/SomeProject" in virtual environments, and "<cwd>/src/SomeProject"
+* For editable installs, the clone location by default is ``<venv
+  path>/src/SomeProject`` in virtual environments, and
+  ``<cwd>/src/SomeProject``
   for global installs.  The :ref:`--src <install_--src>` option can be used to
   modify this location.
 * For non-editable installs, the project is built locally in a temp dir and then
   installed normally. Note that if a satisfactory version of the package is
-  already installed, the VCS source will not overwrite it without an `--upgrade`
-  flag. VCS requirements pin the package version (specified in the `setup.py`
+  already installed, the VCS source will not overwrite it without an
+  ``--upgrade``
+  flag. VCS requirements pin the package version (specified in the ``setup.py``
```

Comment:
This line can be taken up to remove an unnecessary line break.

Reply:
I was doing this to reduce line churn, but will update.